### PR TITLE
9 docs to nist pages

### PR DESCRIPTION
**Pull request by BijalBPatel**
Initial refresh of the docs - involved relaxing requirements.txt

To-dos before merging:

- [x]  Test operation with upgraded dependencies
- [x] Explore Real import of cyrsoxs instead of mock import
- [ ] Explore embedding Jupyter notebooks/ live run with Binder
- [ ] Possibly move tutorials to the user guide section until we have user guide content.

### CODE REVIEW

**Comment by BijalBPatel** [comment]
Possible handy resource for installing conda packages in github actions: 
https://autobencoder.com/2020-08-24-conda-actions/

**Comment by pbeaucage** [comment]
Look at the documentation for NISTtheDocs2Death; it has arguments for a conda environment file and for an apt file in addition to pip

**Comment by pbeaucage** [comment]
One thing to be aware of in working on CI scripts is that they have a _lot_ of access and you need to be exceptionally careful regarding security and software supply chain.  For instance, most GHA scripts have access to the keys used to deploy packages to PyPI, some run on local hardware, etc.  A fairly common attack pattern is to compromise some package used in a chain, then use that exploit together with a PR to a repo to deploy and sign a compromised package that includes whatever nefarious code you want.  Users' machines will merrily pick up this "update" and start running the nefarious code -- because it was signed just like a real update!

There are a lot of best practices for security in CI but the two I would heavily suggest are (1) only use actions or components from people who are extremely well trusted or that you wrote yourself, and (2) always pin the version of CI actions.  This prevents a compromise in one of the actions you use from propagating into your CI chain.  This is nowhere near perfect, but prevents some of the more basic attacks.

Bottom line - any time a blog post tells you to use an action that isn't traceable to a Fortune 50 tech company, start setting off alarm bells in your head.

**Comment by BijalBPatel** [comment]
I think we could go ahead and merge this. 

One thing I'm not too happy about is that the conda install. 

Just feeding the main environment.yml file to NISTthedocs2Death led to it timing out on solving the configuration. 

I implemented a workaround with a separate docs/environment_docs.yml that installs just CyRSoXS with conda, and then pip installs everything else into the same environment. It still takes a long time (~10 mins).

**Review by pdudenas — APPROVED** [review]

**Comment by pdudenas** [comment]
Thanks @BijalBPatel!
